uiowahjmjohnsonselt2024/projectdirectory-selt_2024_team_009
# IMPL: Grid Cells Background Image gen with Open AI DALL-E , inventory, Marketplace

## What does this PR do? 
This PR introduces dynamic image generation for grid cells using OpenAI DALL-E. Additionally, it integrates relevant items into the Marketplace, updates the inventory page background for better display, and rectifies the "Delete Items from Inventory" feature.

## Related Issues 
Relates to #107, #106, #105, #104, #103, #102

## Testing
1. Start the server and navigate to the grid cells page to confirm dynamic image generation.
2. Verify that items appear correctly in the Marketplace.
3. Open the inventory page and ensure the background displays as intended.
4. Test the "Delete Items from Inventory" functionality to confirm it works without issues.

## Checklist

## Extra Information
- Improved the visual aesthetics of the inventory page.
- Enhanced performance when dynamically generating grid cell images.
- Screenshots of the updated inventory page are available for review.
